examples/galaxies/plot_ellipticity.py

```diff
@@ -0,0 +1,128 @@
+"""
+Galaxy Ellipticity Distributions
+================================
+
+This example demonstrate how to sample ellipticity distributions
+in SkyPy.
+
+"""
+
+
+# %%
+# 3D Ellipticity Distribution
+# ---------------------------
+#
+# In Ryden 2004 [1]_, the ellipticity is sampled by randomly projecting
+# a 3D ellipsoid with principal axes :math:`A > B > C`.
+#
+# The distribution of the axis ratio :math:`\gamma = C/A` is a truncated
+# normal with mean :math:`\mu_\gamma` and standard deviation
+# :math:`\sigma_\gamma`.
+#
+# The distribution of :math:`\epsilon = \log(1 - B/A)` is truncated normal
+# with mean :math:`\mu` and standard deviation :math:`\sigma`.
+#
+#
+# The func:`skypy.galaxies.morphology.ryden04_ellipticity()` model samples
+# projected 2D axis ratios. Specifically, it samples the axis ratios of the
+# 3D ellipsoid according to the description above [1] and
```

Comment:
The `[1]` hyperlink only works the first time. I don't think it is needed here though, we shouldn't need to mention "Ryden 2004" multiple times in a single paragraph.

Reply:
isn't only the underscore suffix `[1]_` missing?

Reply:
The underscore only works once and it's already in place (line 15)